src/layers/microscopy-viewer-layer/data-utils/tiff-utils.js

```diff
@@ -0,0 +1,46 @@
+import {fromUrl, Pool, getDecoder } from 'geotiff/dist/geotiff.bundle.min.js';
```

Comment:
Is it something about their packaging that forces us to reference it this way?

Reply:
Yes.  [People](https://github.com/geotiffjs/geotiff.js/issues/98) seem to have trouble with it